Resources/Prototypes/Entities/Objects/Specific/Medical/handheld_crew_monitor.yml

```diff
@@ -31,15 +31,71 @@
   - type: StationLimitedNetwork
   - type: StaticPrice
     price: 500
+
+- type: entity
+  name: cmo's handheld crew monitor
+  parent: HandheldCrewMonitor
+  suffix: DO NOT MAP
+  id: CMOHandheldCrewMonitor
+  components:
   - type: Tag
     tags:
     - HighRiskItem
   - type: StealTarget
     stealGroup: HandheldCrewMonitor
 
 - type: entity
-  id: HandheldCrewMonitorEmpty
+  id: CMOHandheldCrewMonitorEmpty
+  parent: CMOHandheldCrewMonitor
+  suffix: Empty
+  components:
+  - type: ItemSlots
+    slots:
+      cell_slot:
+        name: power-cell-slot-component-slot-name-default
+
+- type: entity
+  name: paramedic handheld crew monitor
   parent: HandheldCrewMonitor
+  id: ParamedicHandheldCrewMonitor
+  description: A paramedic-specific hand-held crew monitor displaying the status of suit sensors. It only shows crew in a critical state or already dead.
+  components:
+  - type: Sprite
+    sprite: Objects/Specific/Medical/handheldcrewmonitor.rsi
+    state: paramedic
+  - type: CrewMonitoringConsole
+    showConsciousListings: false
+    playNoise: true
+
+- type: entity
+  id: ParamedicHandheldCrewMonitorEmpty
+  parent: ParamedicHandheldCrewMonitor
+  suffix: Empty
+  components:
+  - type: ItemSlots
+    slots:
+      cell_slot:
+        name: power-cell-slot-component-slot-name-default
+
+- type: entity
+  name: security handheld crew monitor
+  parent: HandheldCrewMonitor
+  id: SecurityHandheldCrewMonitor
+  description: A security-specific hand-held crew monitor displaying the status of suit sensors.
+  components:
+  - type: Sprite
+    sprite: Objects/Specific/Medical/handheldcrewmonitor.rsi
+    state: security
+  - type: CrewMonitoringConsole
+    trackedDepartments:
+      - Command
+      - Security
+    trackedJobs:
+      - Prisoner
```

Comment:
:trollface: if you used ProtoId youd have already caught this one

Reply:
![image](https://github.com/space-wizards/space-station-14/assets/52761126/a81bd16a-a6b0-4928-9e73-dcde06cf8ff5)

it works, no?